src/solvers.jl

```diff
@@ -195,3 +196,47 @@ function solve(solver::TruncatedSVD, A, y)
     return Dict{String, Any}("C" => solver.P \ θP)
 end
 
+
+@doc raw"""
+`struct ASP` : Active Set Pursuit sparse solver
+    solves the following optimization problem using the homotopy approach:
+
+    ```math 
+    \max_{y} \left( b^T y - \frac{1}{2} λ y^T y \right)
+    ```
+        subject to
+        
+    ```math
+        \|A^T y\|_{\infty} \leq 1.
+    ```
+
+    * Input
+    * `A` : `m`-by-`n` explicit matrix or linear operator.
+    * `b` : `m`-vector.
+    * `min_lambda` : Minimum value for `λ`. Is set to zero if not input is given.
```

Comment:
sorry to be a pain and ask for another change:   A, b are inputs but the remaining "inputs" here are better thought of as solver parameters. I think those should be set when constructing the solver? It could be done quite crudely, e.g. via 
```julia
struct ASP 
   P 
   params
end
```
and then call the homotopy via 
```julia
asp_homotopy(AP, y; solver.params...)
```